### DISCUSSION TITLE
Colour Scheme

### DISCUSSION THREAD

**Discussion by BrodieBrown1998** [gh44922023/Tafe230221#6] [Q&A] [answered]
Is the colour scheme okay, or should it change?
![Colour Scheme](https://github.com/gh44922023/Tafe230221/assets/141377278/dcb39d3c-377a-4e9d-94e6-99b9f5aea36d)

**Comment by jeffrey-tafe** [comment, marked as answer]
Happy to go with this colour scheme.

**Reply by gh44922023** [reply]
Yes all good

**Comment by jeffrey-tafe** [comment]
@gh44922023 Are you cool with this?
